src/test/java/com/openlattice/shuttle/transforms/TransformTest.java

```diff
@@ -24,6 +24,8 @@
         testrow.put("SSN", null);
         testrow.put("Address", "560 Scott Street, San Francisco, CA 94117");
         testrow.put("CommittedDateTime", "03/05/00 10:00");
+        testrow.put("Latt", "36.23452");
+        testrow.put("Long", "30.34573");
```

Comment:
Also, it might make the test a little easier to maintain if you extract the lat/long values into constants up top, and then refer to those constants when writing to / reading from the data. i.e.
```
String lat = "36.23452";
String long = "30.34573";

...

testrow.put("Lat", lat);
testrow.put("Long, long);

...

Assert.assertEquals(lat + "," + long, geographypointTest1);
```

and then if anything gets changed in the future for some reason, the tests won't mysteriously break